web/template/pa-hub/pa-deputy-details.gotmpl

```diff
@@ -17,11 +17,20 @@
         <div class="moj-button-menu">
             <div class="moj-button-menu__wrapper">
                 <a
-                    href="{{ prefix (printf "/%d/change-ecm" .DeputyDetails.ID) }}"
+                    href="{{ sirius "" }}/api/v1/deputies/{{ .DeputyDetails.ID }}/annual-reports?OPG-Bypass-Membrane=1"
```

Comment:
same here, could have just changed `prefix` to `sirius` and fixed the URL format